### ISSUE TITLE
Add team mode

### ISSUE THREAD

**Issue by Tamriel** [huytd/agar.io-clone#66]
maybe even where you can choose which team to join, so playing 6 vs 6 on your local lan party is possible

**Comment by boynet** [comment]
should be pretty easy.. anyone have rules list for team playing?
team joining should be random or something like each new player join the weakest team?
team coloring?(blue red green)
max teams size?(max 3 or 4)

**Comment by Ariamiro** [comment]
to do this we need to use a sepparate server

**Comment by Logan7432** [comment]
yeah. but all we need is merging and viruses... i dont get why this isnt in game already, they get this far into creating this and it dosnt have the key parts

**Comment by abalabahaha** [comment]
@Logan7432 This is a community-based volunteer project. Nobody is getting paid to do this, so nobody has any big incentive to continue developing this (especially as the easier parts of the project are mostly implemented), so the project's gone pretty far already. Currently, the project **is not meant to be a full clone of the original**. Since you seem to think it's easy enough, by all means, go code these features yourself! Or, if you're relying on the community, **be patient**.
